setup_cosim_build_env.sh

```diff
@@ -15,6 +15,7 @@ export USE_QNNPACK=0
 export USE_DISTRIBUTED=0
 export USE_OPENMP=0
 export ATEN_THREADING=NATIVE
+export CFLAGS='-fuse-ld=gold'
```

Comment:
Could you fill us in on why we need to use `gold`?

Reply:
As you might have experienced linking can be bottleneck. Gold is faster linker that available in devtoolset-8 and is compatible with `ld`.

Reply:
Cool!